tests/functional/test_performance.py

```diff
@@ -140,4 +185,40 @@ def test_basic_balance_on_whole_year() -> None:
     status = problem.solver.Solve()
 
     assert status == problem.solver.OPTIMAL
-    assert problem.solver.Objective().Value() == 3000 * horizon
+    assert problem.solver.Objective().Value() == 30 * 100 * horizon
+
+
+def test_basic_balance_on_whole_year_with_large_sum() -> None:
+    """
+    Balance on one node with one fixed demand and one generation with storage, on 8760 timestep.
+    """
+
+    scenarios = 1
+    horizon = 8760
+    time_block = TimeBlock(1, list(range(horizon)))
+
+    database = DataBase()
+    database.add_data("D", "demand", generate_const_data(100, horizon, scenarios))
+
+    database.add_data("G", "p_max", ConstantData(100))
+    database.add_data("G", "cost", ConstantData(30))
+    database.add_data("G", "full_storage", ConstantData(100 * horizon))
+
+    node = Node(model=NODE_BALANCE_MODEL, id="N")
+    demand = create_component(model=DEMAND_MODEL, id="D")
+    gen = create_component(
+        model=GENERATOR_MODEL_WITH_STORAGE, id="G"
+    )  # Limits the total generation inside a TimeBlock
+
+    network = Network("test")
+    network.add_node(node)
+    network.add_component(demand)
+    network.add_component(gen)
+    network.connect(PortRef(demand, "balance_port"), PortRef(node, "balance_port"))
+    network.connect(PortRef(gen, "balance_port"), PortRef(node, "balance_port"))
```

Comment:
How are port connections going to work in YAML files ? Defined in the model ? Defined in the study ?

Reply:
I am not sure. @sylvlecl or @vargastat would know it better